server/sonar-web/src/main/js/apps/webhooks/components/App.tsx

```diff
@@ -41,15 +42,15 @@ export default class App extends React.PureComponent<Props, State> {
 
   componentDidMount() {
     this.mounted = true;
+    this.setState({ loading: true });
     this.fetchWebhooks();
   }
 
   componentWillUnmount() {
     this.mounted = false;
   }
 
-  fetchWebhooks = ({ organization, component } = this.props) => {
-    this.setState({ loading: true });
```

Comment:
I think we don't need because the loading spinner replaces the whole liste of webhooks. And actually the loading animation is inside the Modal (it wont close until the refresh is finished).